lib/client.js

```diff
@@ -807,15 +808,6 @@ MatrixClient.prototype.publicRooms = function(callback) {
  * @return {module:client.Promise} Resolves: TODO
  * @return {module:http-api.MatrixError} Rejects: with an error response.
  */
-MatrixClient.prototype.registerFlows = function(callback) {
```

Comment:
Not really - you just call register and if it fails, it's in the response, so it's just part of the register call.